### ISSUE TITLE
configuration fails

### ISSUE THREAD

**Issue by samfux84** [gmarcais/Jellyfish#138]
```
[sfux@eu-c7-065-01 Jellyfish-2.2.10]$ autoreconf -fi
libtoolize: putting auxiliary files in `.'.
libtoolize: copying file `./ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
configure.ac:9: error: possibly undefined macro: AC_LIB_RPATH
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:32: error: possibly undefined macro: AC_LIB_LINKFLAGS_FROM_LIBS
autoreconf: /usr/bin/autoconf failed with exit status: 1
```

**Comment by gmarcais** [comment]
Please, download the file `jellyfish-2.2.10.tar.gz`, not `Source code`. It contains the source properly packaged, and you don't need to run `autoreconf` at all. Just run `./configure && make && make install`.

Now, if you really want to start from the git tree (say to modify the code of Jellyfish, rather than using the software), then you need the `gettext` package to properly run `autoreconf`.

**Comment by samfux84** [comment]
Thank you. I downloaded the package that you recommend and the installation worked.


--
ETH Zurich, Dr. sc. ETH Samuel Fux
High Performance Computing, Scientific IT Services
WEC D 13, Weinbergstrasse 11, 8092 Zurich, Switzerland
Phone +41 44 632 57 63, http://www.id.ethz.ch/
samuel.fux@id.ethz.ch
________________________________
Von: gmarcais <notifications@github.com>
Gesendet: Mittwoch, 13. Februar 2019 17:40:55
An: gmarcais/Jellyfish
Cc: Fux Samuel (ID SIS); Author
Betreff: Re: [gmarcais/Jellyfish] configuration fails (#138)


Closed #138<https://github.com/gmarcais/Jellyfish/issues/138>.
